tests/linux/dmidecode_-t_processor/linux_dmidecode_-t_processor.raw

```diff
@@ -0,0 +1,58 @@
+# dmidecode 1.1
+SMBIOS 1.1 present.
+
+Handle 0x0004, DMI type 4, 42 bytes
+Processor Information
+	Socket Designation: CPU 1   
+	Type: Central Processor
+	Family: Core 2 Quad
+	Manufacturer: AMD
```

Comment:
How can the manufacturer be AMD and yet the processor be an Intel one?!
:sweat_smile: